reporter/src/main/kotlin/reporters/TableReporter.kt

```diff
@@ -163,10 +173,20 @@ abstract class TableReporter : Reporter() {
         val scanRecord = ortResult.scanner!!.results
 
         val projectTables = analyzerResult.projects.associate { project ->
+            val projectExclude = ortResult.repository.config.excludes?.findProjectExclude(project)?.let {
+                // Only add the project exclude to the model if the whole project is excluded. If only parts of the
+                // project are excluded this information will be stored in the rows of the affected dependencies.
+                if (it.exclude) it else null
```

Comment:
Shorter: `it.takeIf { it.exclude }`. But then the outer `it` should probably be named so it does not shadow the inner one.

Reply:
Done.